mmv1/products/sql/api.yaml

```diff
@@ -504,6 +504,14 @@ objects:
         description: |
           The name of the database in the Cloud SQL instance.
           This does not include the project ID or instance name.
+      - !ruby/object:Api::Type::String
+        name: 'deletion_policy'
+        description: |
+          The deletion policy for the database. Setting ABANDON allows the resource 
+          to be abandoned rather than deleted. This is useful for Postgres, where databases cannot be 
+          deleted from the API if there are users other than cloudsqlsuperuser with access. Possible 
+          values are: "ABANDON".
+        url_param_only: true  
```

Comment:
Can you instead move this field to a virtual_field type rather than marking as url_param_only?

It is more clear then that the field isn't used in the URL or body of the request

Reply:
They have similar functionality to url_param_only in that they aren't included in the request body, aren't read from the response etc

Reply:
done